tffconnect_frontend/src/app/contexts/JWTAuthContext.js

```diff
@@ -106,14 +106,25 @@ export const AuthProvider = ({ children }) => {
         })
     }
     
-    const register = async (email, username, password) => {
-        const response = await axios.post('/api/auth/register', {
-            email,
+    const register = async (username, first_name, last_name, password) => {
+        const response = await axios.post(API_URL + '/users/signup/', {
             username,
+            first_name,
+            last_name,
             password,
         })
+        .catch((error) => {
+            console.log(error)
+        })
+        console.log(response.data)
+
+        const accessToken = response.data["token"]
```

Comment:
The response from `api/users/signup/` does not include a token. Are you sure your fix works? You must also use the `api/users/login` to obtain the token of the newly created token.